admin/src/main/resources/io/buoyant/admin/js/src/delegator.js

```diff
@@ -67,6 +67,9 @@ define([
           obj.tree.transformation = obj.name;
           obj.tree.isTransformation = true;
           obj.bound.transformed = renderNode(obj.tree, primary);
+          if (ttype != success) {
+            obj.bound.addr.type = "neg";
+          }
```

Comment:
for my own edification, why is this needed? I figured a neg was a neg transformer or no